GOPHERCON_COMPETITION.md

```diff
@@ -0,0 +1,55 @@
+# Gno.land Platform Raffle Competition
+
+Welcome to our Gnolang platform raffle, designed to put our platform to the test!
+This competition is a unique blend of two main areas:
+
+1. **Chess Mastery (50% of Points)**:
+Dive into the strategic world of chess, whether you're a grandmaster or a beginner.
+Every game, every win, and every move you make offers you points.
+
+3. **Platform Engagement (50% of Points)**:
+Outside the chessboard, we invite you to engage with our blockchain, craft integrations, report bugs, and more.
+This segment isn't just for developers; it's for anyone willing to explore and contribute.
+
+Our intent is clear: everyone should have a fair shot at winning.
+This isn't just about expertise in one domain; it's about passion, exploration, and collaboration.
+While we've worked hard on this, please remember, this is an experimental phase, and there might be hiccups along the way.
+We apologize in advance for any bugs you might encounter and truly appreciate your patience and resilience. 
+
+Join us on this adventure! We eagerly await the discussions, feedback, and innovations that will stem from this competition.
+
+## 🔍 Point Breakdown**
+
+1. **Chess Games Completion**: 
+   - Earn 100 points for every finished game.
+   - Max limit: 200 games.
+   - Potential Total: 20,000 points.
+
+2. **Chess Game Wins**: 
+   - Score an additional 400 points for each win.
+   - Max limit: 50 wins.
+   - Potential Total: 20,000 points.
+
+3. **Game Moves**: 
+   - 1 point for each move made.
+   - Max limit: 10,000 moves.
+   - Potential Total: 10,000 points.
+
+4. **Workshop Participation**: 
+   - Flat 20,000 points for joining.
+
+5. **Blockchain Interactions**: 
+   - Delve deeper with:
+      - Reporting bugs.
+      - Making valuable PRs.
+      - Developing and sharing a custom client on GitHub, listing it on our main repo.
+      - Any significant contribution made in tandem with our staff during GopherCon.
+   - Note: Contributions undergo our staff's review. Get recognized early!
+   - Max limit: Depends on quality and quantity.
+   - Potential Total: Up to 30,000 points.
+
+## 🎉 The Grand Raffle:
+
+Once the competition concludes, we'll hold the grand raffle.
+Your accumulated points determine your chances of winning.
+The more points you have, the higher your chances!
```

Comment:
Specify that we'll publish a realm who will get the players from the chess server realm and determine the raffle, pseudorandom, blockchain-verifiable!